src/phylum/ci/ci_github.py

```diff
@@ -171,6 +171,16 @@ def phylum_comment_exists(self) -> bool:
         """Predicate for detecting whether a Phylum-generated comment exists."""
         return bool(get_most_recent_phylum_comment_github(self.comments_url, self.github_token))
 
+    @property
+    def repo_url(self) -> Optional[str]:
+        """Get the repository URL for reference in Phylum project metadata."""
+        # Ref: https://docs.github.com/actions/learn-github-actions/variables#default-environment-variables
+        server_url = os.getenv("GITHUB_SERVER_URL")
+        repo = os.getenv("GITHUB_REPOSITORY")
+        if server_url is None or repo is None:
```

Comment:
It _might_ make sense to log out which are missing just so we don't have to do extra guesswork if this is ever an issue. Probably doesn't need extra logging in the other implementations since they all operate off of one environment variable.